src/i18n/locales/en-US.json

```diff
@@ -238,7 +238,8 @@
       "backButton": "Back",
       "backButton-tooltip": "Return to subtitle selection",
       "editTitle": "Subtitle Editor - {{title}}",
-      "editTitle-loading": "Loading"
+      "editTitle-loading": "Loading",
+		"generic": "Generic"
```

Comment:
```suggestion
	"generic": "Generic"
```

Reply:
That shouldn't have happened. Sorry. Fixed.